src/main/java/frc/robot/subsystems/Manager.java

```diff
@@ -173,6 +195,8 @@ public void periodic() {
                 speedUpTimer.start();
                 intake.setState(IntakeStates.INTAKING);
                 if (speedUpTimer.get() > Constants.Intake.SPINNING_UP_INTAKE_TIME) {
+                    speedUpTimer.stop();
```

Comment:
Wait. If you're in the start spinning state, wouldn't this cause the intake to spin forever, because the timer would keep getting reset?

Reply:
Ig, but it's in auto so it would just mean the note gets spammed centered. I'll remove the reset tho

Reply:
Sounds good, just make sure it's getting reset somewhere else so it doesn't break. It probably wouldn't hurt but you never know what worlds refs are going to be like, and we could damage a note.